EMML/1001-2000/EMML1939.xml

```diff
@@ -35,51 +35,268 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
+</msItem>
+
+<msItem xml:id="p1_i1.10.2">
+<locus from="162r" to="168r"/>
+<title xml:lang="en" type="complete">The death of Joseph</title>
+<incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ መሓሪ፡ ወመስተሣህል፨ ኮነ፡ በውእቱ፡ ዘመን፡ ሶበ፡ በጽሐ፡ መዋዕሊሁ፡ ለዮሴፍ፡ ከመ፡ ይሖር፡ እምዝንቱ፡ ዓለም፡ ኀበ፡ አበዊሁ፡
+ከመ፡ ኵሉ፡ ሰብእ፨</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msContents>
+
+<physDesc>
+<handDesc>
+<handNote script="Ethiopic" xml:id="h1"></handNote>
+</handDesc>
+</physDesc>
+
+</msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+<msItem xml:id="ms_i2">
+<locus from="169r" to="174v"/>
+<title xml:lang="en" type="complete" ref="LIT1700Judges">Judges 1,21-5,26</title>
+</msItem>
+</msContents>
+
+
                <physDesc>
-                  <objectDesc form="Codex">
-                     <supportDesc>
-                        <support>
-                           <material key="parchment"/>
-                        </support>
-                        <extent>
-                           <measure unit="leaf">174 </measure>
-                           <dimensions>
-                              <dim>28 x 20 cm.</dim>
-                           </dimensions>
-                        </extent>
-                        <foliation/>
-                        <condition key="good"/>
-                     </supportDesc>
-                  </objectDesc>
+
+
                   <decoDesc>
                      <decoNote xml:id="d1">
                         <desc>Decorative designs, ff. 2a, 24a, 49a, 60a, 67b, 84a, 86b, 102a, 114a, 124a.</desc>
                      </decoNote>
                   </decoDesc>
-               </physDesc>
-               <history>
-                  <origin>
-                     <origDate notBefore="1300" notAfter="1500">14th-15th century,
-                                                        </origDate>
-                  </origin>
-                  <provenance>In Monastery of Ḥayq Esṭifānos when filmed.</provenance>
-               </history>
-               <additional>
-                  <adminInfo>
-                     <recordHist>
-                        <source>
-                           <listBibl type="catalogue">
-                              <bibl>
-                                 <ptr target="bm:EMML5"/>
-                              </bibl>
-                           </listBibl>
-                        </source>
-                     </recordHist>
-                  </adminInfo>
-               </additional>
+  <additions>
+  <list>
+  <item xml:id="a1">
+  <locus from="1r" to="1v"/>
+  <desc type="GuestText">Fragment from <title ref="LIT1461Gadlah">Gadla Ḥawāryāt</title></desc>
+  </item>
+
+  <item xml:id="a2">
+  <locus from="9r" to="16r"/>
+  <locus target="#17v"/>
+  <locus target="#18r"/>
+  <locus target="#19v"/>
+  <locus from="20r" to="20v"/>
+  <locus from="21v" to="22v"/>
+  <locus from="23r" to="23r"/>
+  <desc type="MalkeHymn">Image of the crucifixion of Christ</desc>
+  </item>
```

Comment:
Is it written in the margins? You could add this information so that the locus becomes clear